bc_obps/registration/fixtures/real/naicsCode.json

```diff
@@ -0,0 +1,230 @@
+[
+  {
+    "model": "registration.naicsCode",
+    "pk": 1,
+    "fields": {
+      "id": 1,
+      "naics_code": "211110",
+      "naics_description": "Oil and gas extraction (except oil sands)"
+    }
+  },
+  {
+    "model": "registration.naicsCode",
+    "pk": 2,
+    "fields": {
+      "id": 2,
+      "naics_code": "212114",
+      "naics_description": "Bituminous coal mining"
+    }
```

Comment:
these two are the only codes with `pk` and `id`, and all the other ones seem to have gone into the db fine, so it looks like we don't have to specify these manually